pkg/sql/opt/optbuilder/select.go

```diff
@@ -441,38 +445,24 @@ func (b *Builder) buildScan(
 	}
 
 	outScope = inScope.push()
+
```

Comment:
Should we check now if the the input set of ordinals is nil? Or could that happen?